projects/fsastorefrontlib/src/cms-components/assets-table/resolve-asset-value.pipe.ts

```diff
@@ -0,0 +1,33 @@
+import { Pipe, PipeTransform } from '@angular/core';
```

Comment:
This pipe also needs to be exported in index.ts. It appears that's the reason Travis is failing